src/services/HttpService.ts

```diff
@@ -8,15 +8,15 @@ export interface HttpService {
   get<T>(
     url: string,
     queryParams: QueryParams,
-    clientSdk: ClientSdk,
+    clientSdk: Record<string, string>,
     authToken?: string
   ): Promise<T>;
 
   post<T>(
     url: string,
     queryParams: QueryParams,
     body: QueryParams,
-    clientSdk: ClientSdk,
+    clientSdk: Record<string, string>,
```

Comment:
since we mostly keep get/post methods to be pretty generic, would it make sense to pass something like a headerParams or additionalHeaderParams which includes `Client-SDK` instead of passing it directly in as an argument?

Reply:
If you remember we ran into an issue where we weren't passing cookies in requests. After that, I realized that we were trying to make HttpService too generic, more generic than it needed to be. I think it will be very easy to refactor this to handle multiple headers if we need to, but that's a long way off. This also prevents people from cheating additional headers into the request by using the library through javascript, since we won't be doing a loop over the additional header values but just picking exactly the one we want, Client-SDK.

Reply:
Since you might be curious, here's the PR in question https://github.com/yext/answers-core/pull/97